src/components/clusterConfiguration/ClusterConfiguration.tsx

```diff
@@ -112,10 +131,14 @@ const findMatchingSubnet = (
   return matchingSubnet || 'No subnets available';
 };
 
+type ClusterConfigurationProps = {
+  cluster: Cluster;
+};
+
 const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster }) => {
+  const [submitType, setSubmitType] = React.useState('save');
```

Comment:
this looks weird to me. Why not just pass the submit type to `handleSubmit` func instead ?

Reply:
submitType also drives the variant of validationSchema prop, so I had to push the state for this to the component level.